examples/kubernetes/hpe-nimble-storage/README.md

```diff
@@ -62,12 +62,12 @@ Cloning supports two modes of cloning. Either use `cloneOf` and reference a PVC
 | createSnapshot | Boolean | Indicates that a new snapshot of the volume should be taken matching the name provided in the `snapshot` parameter. If the `snapshot` parameter is not specified, a default name will be created. |
 
 ### Import parameters
-Importing volumes to Kubernetes requires the source Nimble volume to be offline. All previous Access Control Records and Initiator Groups will be stripped from the volume when put under control of the HPE CSI Driver.
+Importing volumes to Kubernetes requires the source Nimble volume to be offline. In case of reverse replication, the upstream volume should not have any active sessions and to be in offline state. All previous Access Control Records and Initiator Groups will be stripped from the volume when put under control of the HPE CSI Driver.
 
 | Parameter | String | Description |
 | --------- | ------ | ----------- |
 | importVolumeName | Text | The name of the Nimble volume to import. |
 | snapshot | Text | The name of the Nimble snapshot to restore the imported volume to after takeover. If not specified, the volume will not be restored. |
-| takeover | Boolean | Indicates the current group will takeover ownership of the Nimble volume and volume collection. This should be performed against a downstream replica. |
-| reverseReplication | Boolean | Reverses the replication direction so that writes to the Nimble volume are replicated back to the group where it was replicated from. |
+| takeover | Boolean | Indicates the current group will takeover ownership of the Nimble volume and volume collection. This should be performed against a downstream replica and the upstream volume should not have active sessions with initiators. |
+| reverseReplication | Boolean | Reverses the replication direction so that writes to the Nimble volume are replicated back to the group where it was replicated from. The upstream volume should not have active sessions with initiators. |
```

Comment:
I don't think this change is necessary.  The volume's required state is already stated above.

Reply:
Agreed, will remove.